DuckDuckGo/NetworkProtection/AppTargets/BothAppTargets/VPNUIActionHandler.swift

```diff
@@ -25,7 +25,7 @@ import VPNAppLauncher
 
 /// Main App's VPN UI action handler
 ///
-final class VPNUIActionHandler: VPNUIActionHandling {
```

Comment:
Moving some code around to make the class easier to follow.  This is moved to an extension below.